src/OpenSage.Game/Scene3D.cs

```diff
@@ -423,7 +423,7 @@ internal void BuildRenderList(RenderList renderList, Camera camera, in TimeInter
         // This is for drawing 2D elements which depend on the Scene3D, e.g tooltips and health bars.
         internal void Render(DrawingContext2D drawingContext)
         {
-            _scene25D.Draw(drawingContext);
+            _scene25D?.Draw(drawingContext);
```

Comment:
Unrelated to animation scaling - I hit a crash when opening the developer mode Preview window.